scan.py

```diff
@@ -352,8 +352,8 @@ def scan_dataset(project, ds_id, ds_path, mode, location):
     expected_facets = options.facet_rules[project]
 
     try:
-        character = extract_character(nc_files, mode, location, var_id=facets['variable'],
-                                      expected_attrs=expected_facets)
+        character = extract_character(nc_files, location, var_id=facets['variable'],
+                                      mode=full, expected_attrs=expected_facets, )
```

Comment:
@ellesmith88 , just spotted `mode=full` here, wanted to check that `full` is being defined as a variable. If not, should it be `mode=mode` or `mode='full'`?